tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -177,34 +175,35 @@ def cmd(self):
     def pre_checks(self):
         return ["requires_openssl_tls1_3"]
 
-    def post_checks(self):
-        return []
-
-    def pre_cmd(self):
-        return []
-
-    def hrr_post_checks(self, named_group):
-        return []
-
 
 class OpenSSLServ(OpenSSLBase):
     """
     Generate test commands for OpenSSL server.
     """
 
     def cmd(self):
-        ret = self.pre_cmd() + super().cmd()
-        ret += ['-accept $SRV_PORT']
-
+        ret = super().cmd()
         ret += ['-num_tickets 0 -no_resume_ephemeral -no_cache']
-
-        return ' '.join(ret)
+        return ret
 
     def post_checks(self):
         return ['-c "HTTP/1.0 200 ok"']
 
     def pre_cmd(self):
-        return ['$O_NEXT_SRV_NO_CERT']
+        ret = ['$O_NEXT_SRV_NO_CERT']
+        for _, cert, key in map(lambda sig_alg: CERTIFICATES[sig_alg], self._cert_sig_algs):
```

Comment:
Why move the share code to the sub class? seems define it in OpenSSLBase can save code size.

Reply:
Client side and server side use different parameter for this parameter. 
For server side , we should add certificate files. For client side, we should add cafile. 
We should not add both of them.

Reply:
But seems for this code : "for _, cert, key in map(lambda sig_alg: CERTIFICATES[sig_alg], self._cert_sig_algs):", we can share it. Anyway, it's fine for me to put it in the sub class.